modules/ROOT/pages/cookies.adoc

```diff
@@ -0,0 +1,158 @@
+= Configure Cookies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Anypoint Exchange uses cookies to save user settings, enhance user experience, and improve performance.
+
+Exchange requires some cookies to work correctly. These include cookies for session details and authentication information.
+
+Other functional cookies are not required, although certain functionality does not work without them. These include cookies for site analytics and advertising.
```

Comment:
Functional cookies are not needed for the website to work correctly. It's only the advertising cookies, which we think may interfere with some functionality such as videos.

Reply:
I updated the text and corrected the information about the functional cookies.